addons/crm_livechat/models/mail_channel.py

```diff
@@ -26,16 +26,25 @@ def _execute_command_lead(self, **kwargs):
         self._send_transient_message(partner, msg)
 
     def _convert_visitor_to_lead(self, partner, channel_partners, key):
+        """ Create a lead from channel /lead command
+        :param partner: internal user partner (operator) that created the lead;
+        :param channel_partners: channel members;
+        :param key: operator input in chat ('/lead Lead about Product')
+        """
         description = ''.join(
             '%s: %s\n' % (message.author_id.name or self.anonymous_name, message.body)
             for message in self.channel_message_ids.sorted('id')
         )
+        # filter out partners linked to a public user; easier than a search to avoid
+        # domain using user_ids.groups_id
+        customers = channel_partners.partner_id.filtered(lambda p: p != partner and (not p.user_ids or any(not user._is_public() for user in p.user_ids)))
```

Comment:
I wonder, if an operator is inviting another operator onto the chat, shouldn't we also exclude other operators from this?